test/db/cmd/cmd_open

```diff
@@ -410,36 +410,53 @@ RUN
 NAME=o-*
 FILE==
 CMDS=<<EOF
-o -
```

Comment:
I am surprised it was working before.

Reply:
It tested that after `o-*` everything was deleted... but it never said that before it anything was loaded :-)